Orm/Xtensive.Orm.Manual/Prefetch/PrefetchTest.cs

```diff
@@ -236,7 +235,7 @@ public async Task MultipleBatchesAsyncTest()
 
       int count = 1000;
 
-      using (var session = domain.OpenSession())
+      await using (var session = await domain.OpenSessionAsync())

```

Comment:
```suggestion
      var session = await domain.OpenSessionAsync().ConfigureAwait(false);
      await using (session.ConfigureAwait(false))
```

Reply:
There is o need to do this in test code in case we don't synchronously wait for async method completion.